### ISSUE TITLE
Pop-up message saying somthn like "we'll ask you again to confirm your PIN in xy days/weeks" still untranslated

### ISSUE THREAD

**Issue by bungabunga** [signalapp/Signal-Android#10236]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Every once in a while I have to re-confirm my PIN I get a pop up message saying somthn like "we'll ask you again to confirm your PIN in xy days/weeks", which is still untranslated. As a translator I remember that I already translated this string quite some time ago but on device it's still untranslated.

### Steps to reproduce
- wait for device to ask you to confirm your PIN
- input your PIN
- the untranslated pop-up message appears

### Device info
**Device:** Pixel 3
**Android version:** 11
**Signal version:** 4.79.1 (that happened on previous versions, not able to reproduce whenever I want)
**Language:** Slovenian

**Comment by greyson-signal** [comment]
We have those translations in the app. How have you set your language? In your system settings, or in your Signal-specific settings? Are other things in the app unexpectedly untraslated?

**Comment by bungabunga** [comment]
Hey, I set my language in Signal settings, bacause I prefer system language to be English. No sudden untranslated stuff in Signal. Also, this is not a new thing, it's untranslated from the introduction of PINs.

I just searched Transifex: Signal strings for Slovenian language are 100 translated, but I couldn't find that specific string. I found the ones for wrong PIN and ones saying PIN would be mandatory soon, but not that one.
